resources/user-agents/crawlers/rippers/python-requests.json

```diff
@@ -1,149 +1,47 @@
+  "division": "python-requests #MAJORVER#.#MINORVER#",
+  "versions": ["2.18","2.17","2.16","2.15","2.14","2.13","2.12","2.11","2.10","2.9","2.8","2.7","2.6","2.5","2.4","2.3","2.2","2.1","2.0","1.2","1.1"],
   "sortIndex": 481,
   "lite": false,
   "standard": true,
   "userAgents": [
     {
-      "userAgent": "python-requests",
+      "userAgent": "python-requests #MAJORVER#.#MINORVER#",
       "properties": {
         "Parent": "DefaultProperties",
-        "Comment": "python-requests",
+        "Comment": "python-requests #MAJORVER#.#MINORVER#",
         "Browser": "python-requests",
         "Browser_Maker": "Python Software Foundation",
         "Browser_Type": "Bot/Crawler",
+        "Version": "#MAJORVER#.#MINORVER#",
+        "MajorVer": "#MAJORVER#",
+        "MinorVer": "#MINORVER#",
         "Crawler": "true"
       },
       "children": [
         {
-          "match": "python-requests/2.5* CPython/#PLATFORM#",
-          "device": "Linux Desktop",
-          "properties": {
-            "Version": "2.5",
-            "MajorVer": "2",
-            "MinorVer": "5"
-          },
-          "platforms": [
-            "Linux_64", "Linux_32", "Linux"
-          ]
-        },
-        {
-          "match": "python-requests/2.5*",
-          "properties": {
-            "Version": "2.5",
-            "MajorVer": "2",
-            "MinorVer": "5"
-          }
-        },
-        {
-          "match": "python-requests/2.4* CPython/#PLATFORM#",
-          "device": "Linux Desktop",
-          "properties": {
-            "Version": "2.4",
-            "MajorVer": "2",
-            "MinorVer": "4"
-          },
-          "platforms": [
-            "Linux_64", "Linux_32", "Linux"
-          ]
-        },
-        {
-          "match": "python-requests/2.4*",
-          "properties": {
-            "Version": "2.4",
-            "MajorVer": "2",
-            "MinorVer": "4"
-          }
-        },
-        {
-          "match": "python-requests/2.3* CPython/#PLATFORM#",
-          "device": "Linux Desktop",
-          "properties": {
-            "Version": "2.3",
-            "MajorVer": "2",
-            "MinorVer": "3"
-          },
-          "platforms": [
-            "Linux_64", "Linux_32", "Linux"
-          ]
+          "match": "python-requests/#MAJORVER#.#MINORVER#*"
         },
         {
-          "match": "python-requests/2.3*",
-          "properties": {
-            "Version": "2.3",
-            "MajorVer": "2",
-            "MinorVer": "3"
-          }
-        },
-        {
-          "match": "python-requests/2.2* CPython/#PLATFORM#",
+          "match": "python-requests/#MAJORVER#.#MINORVER#* CPython/#PLATFORM#",
           "device": "Linux Desktop",
-          "properties": {
-            "Version": "2.2",
-            "MajorVer": "2",
-            "MinorVer": "2"
-          },
           "platforms": [
-            "Linux_64", "Linux_32", "Linux"
+            "Linux_64","Linux_64_amd", "Linux_32", "Linux"
```

Comment:
Usually you can remove "Linux_32" as the "Linux" platform will catch anything that the "Linux_64" and "Linux_64_amd" platforms don't (and has the same properties defined as "Linux_32".